### DISCUSSION TITLE
Error happened "CMake Error at CMakeLists.txt:42 (message):   32bit platforms are not supported" when compiling the source code at the MAC book

### DISCUSSION THREAD

**Discussion by TonyLv** [apache/doris#38398] [A - General / Q&A]
The error snapshot is at the following:
<img width="1136" alt="image" src="https://github.com/user-attachments/assets/a6d49289-3fb7-456a-b3ae-1e9dbfc3efa3">

Anyone has the solution about that? Thanks

**Comment by ixzc** [comment]
Is your machine 32bit?

**Comment by zy-kkk** [comment]
The cloud module does not support Mac compilation yet. You can specify --fe --be after build.sh to skip the compilation of the cloud module.
